_shared/project/tox.ini

```diff
@@ -30,6 +30,7 @@ setenv =
     dev: NEW_RELIC_APP_NAME = {env:NEW_RELIC_APP_NAME:{{ cookiecutter.slug }}}
     dev: NEW_RELIC_ENVIRONMENT = {env:NEW_RELIC_ENVIRONMENT:dev}
 {% if cookiecutter.get("postgres") == "yes" %}
+    dev: ALEMBIC_CONFIG = {env:ALEMBIC_CONFIG:{{ cookiecutter.package_name }}/migrations/alembic.ini}
```

Comment:
Is this not in conf/ instead of migrations/?

https://github.com/hypothesis/cookiecutters/blob/962060d3f62145840ea299a9a95c3ce706d2413d/_shared/project/conf/alembic.ini

Reply:
D'oh, yep, well spotted. Fixed